library/tasks/Picard.wdl

```diff
@@ -1,3 +1,43 @@
+task SortBam {
+    File bam_input
+    String sort_order = "coordinate"
+
+    # runtime values
+    String docker = "quay.io/humancellatlas/secondary-analysis-picard:v0.2.2-2.10.10"
+    Int machine_mem_mb = 7500
+    Int machine_overhead_mb = 500
+    Int command_mem_mb = machine_mem_mb - machine_overhead_mb
+    Int cpu = 1
+    Int disk = ceil(size(bam_input, "G") * 6) + 50
+    Int preemptible = 0
```

Comment:
Not sure why this would be non-preemptible. Does it typically take more than 2 hours to run?

Reply:
No it takes 20 min max so far. I thought we are making everything pre-emptible at the pipeline level in another ticket. Shall I set it to 1 or shall remove pre-emptible requirement.

Reply:
I'd prefer to keep `0` here since we could always override this value in the optimus pipeline level, plus we have this ticket to evaluate the tasks of Optimus and optimize them altogether: https://app.zenhub.com/workspaces/dcp-backlogs-5ac7bcf9465cb172b77760d9/issues/humancellatlas/secondary-analysis/140